packages/blade/src/utils/getStringChildren/getStringChildren.ts

```diff
@@ -1,5 +1,5 @@
-import { isEmpty } from 'lodash';
```

Comment:
We probably need lodash esm or webpack lodash plugin if we don't want to do `'lodash/isEmpty'`

Reply:
yup. I think we're anyway re-exporting it. auto-import imported it and I didn't notice earlier ;__;

We can have some eslint rules to disable these imports.